devops/openshift/web-deploy.yml

```diff
@@ -105,9 +112,9 @@ parameters:
   - name: SERVICE_NAME
     value: web
   - name: CPU_LIMIT
-    value: "0.5"
+    value: "0.25"
   - name: MEMORY_LIMIT
-    value: "256M"
+    value: "512M"
```

Comment:
SIMS-Api will do way more processing than the Web POD, why we would justify more memory allocation to Web than API?

Reply:
initially we had only one pod API with more memory, but due to the change in replication controller to have max replicas of 10 we changed it to smaller numbers. In the case of web, even though there are more processing done by API, the web had to render the application faster, so for safer side, I had the values in the vertical pod bumped up.

Reply:
The POD will not render anything, it will just allow the download of static files (HTML, js, CSS, ect.). I did not follow the explanation.